src/package.ps1

```diff
@@ -102,6 +102,14 @@ function AsPackage {
 	throw "failed to parse package: $Pkg"
 }
 
+function ResolvePackageRefPath {
+	param (
+		[Parameter(Mandatory, ValueFromPipeline)]
+		[object]$Pkg
```

Comment:
```suggestion
		[Collections.Hashtable]$Pkg
```

Will you update `ResolveRemoteRef`'s param as well?

Reply:
Done. Yeah, I wasn't sure which one it was supposed to be, since some were `[object]` and some were `[Collections.Hashtable]`.